eiffel-vocabulary/EiffelIssueDefinedEvent.md

```diff
@@ -69,52 +66,39 @@ This section describes which link types are valid for this event type. For detai
 __Required:__ No  
 __Legal targets:__ Any  
 __Multiple allowed:__ Yes  
-__Description:__ Identifies a cause of the event occurring. SHOULD not be
-used in conjunction with __CONTEXT__: individual events providing __CAUSE__
-within a larger context gives rise to ambiguity. It is instead recommended to
-let the root event of the context declare __CAUSE__.
+__Description:__ Identifies a cause of the event occurring. SHOULD not beused in conjunction with __CONTEXT__: individual events providing __CAUSE__within a larger context gives rise to ambiguity. It is instead recommended tolet the root event of the context declare __CAUSE__.
```

Comment:
```suggestion
__Description:__ Identifies a cause of the event occurring. SHOULD not be used in conjunction with __CONTEXT__: individual events providing __CAUSE__ within a larger context gives rise to ambiguity. It is instead recommended to let the root event of the context declare __CAUSE__.
```

Reply:
Fixed in 837dc1f55476b3ca0ba2d9e8c235f39c4cd7ecf3.